util/addDataFromZib/addDataFromZib.xsl

```diff
@@ -20,6 +20,146 @@
+            </xsl:choose>
+            <!-- Automagical description takes too much time atm. -->
+            <!--<xsl:choose>
+                <xsl:when test="not(f:description)">
+                    <publisher value="Nictiz"/>
+                </xsl:when>
+                <xsl:otherwise>-->
+            <xsl:apply-templates select="f:description"/>
+            <!--</xsl:otherwise>
+            </xsl:choose>-->
+            
+            <xsl:apply-templates select="f:useContext | f:jurisdiction"/>
+            
+            <!-- Add purpose, copyright -->
+            <xsl:choose>
+                <xsl:when test="not(f:purpose) and starts-with($id, 'zib-')">
+                    <xsl:variable name="mapping" select="f:mapping[f:identity/starts-with(@value, 'zib-')][1]"/>
+                    <purpose value="This {f:type/@value} resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) {replace($mapping/f:name/@value, 'zib ','')}]({$mapping/f:uri/@value})."/>
+                </xsl:when>
+                <xsl:when test="not(f:purpose) and starts-with($id, 'nl-core-')">
+                    <purpose value="A derived profile from [{replace($id, 'nl-core-','zib-')}](http://nictiz.nl/fhir/StructureDefinition/{replace($id, 'nl-core-','zib-')}) to provide a version better suited for implementation purposes. This profile augments the base profile with elements found in the various use cases that have adopted the zib."/>
+                </xsl:when>
+                <xsl:otherwise>
+                    <xsl:apply-templates select="f:purpose"/>
+                </xsl:otherwise>
+            </xsl:choose>
+            <xsl:choose>
+                <xsl:when test="not(f:copyright)">
+                    <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise."/>
+                </xsl:when>
+                <xsl:otherwise>
+                    <xsl:apply-templates select="f:copyright"/>
+                </xsl:otherwise>
+            </xsl:choose>
+            
+            <xsl:apply-templates select="f:keyword | f:fhirVersion | f:mapping | f:kind"/>
+            
+            <xsl:choose>
+                <xsl:when test="f:abstract/@value = 'false' and starts-with($id, 'zib-')">
+                    <abstract value="true"/>
+                </xsl:when>
+                <xsl:otherwise>
+                    <xsl:apply-templates select="f:abstract"/>
+                </xsl:otherwise>
+            </xsl:choose>
+            
+            <xsl:apply-templates select="f:context | f:contextInvariant | f:type"/>
+            
+            <xsl:choose>
```

Comment:
Klein wenkbrauwfronsje hier. Je kan in Forge je zib-profiel aanklikken en "Derive from" kiezen.

Reply:
(Zit een typo in zie ik)
Klopt, zo stond het eerst - maar zelfs dat voor user A. Toonstra 'te veel' moeite :D. En op zich heeft hij een punt dat dit weer een paar kliks in Forge scheelt.

Reply:
Dan vrees ik toch dat ik het gedocumenteerd wil zien voordat ik mijn zegen geef.